src/client/datascience/jupyter/kernels/kernelSelector.ts

```diff
@@ -1,54 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { inject, injectable } from 'inversify';
-import { IApplicationShell, ICommandManager, IVSCodeNotebook } from '../../../common/application/types';
+import { ICommandManager, IVSCodeNotebook } from '../../../common/application/types';
 import '../../../common/extensions';
 import { Resource } from '../../../common/types';
-import * as localize from '../../../common/utils/localize';
-import { Commands } from '../../constants';
 import { IInteractiveWindowProvider } from '../../types';
-import { getDisplayNameOrNameOfKernelConnection } from './helpers';
-import { KernelConnectionMetadata } from './types';
 import { getActiveInteractiveWindow } from '../../interactive-window/helpers';
-import { IServiceContainer } from '../../../ioc/types';
 import { getResourceType } from '../../common';
 import { traceError } from '../../../common/logger';
 
-/**
- * All KernelConnections returned (as return values of methods) by the KernelSelector can be used in a number of ways.
- * E.g. some part of the code update the `interpreter` property in the `KernelConnectionMetadata` object.
- * We need to ensure such changes (i.e. updates to the `KernelConnectionMetadata`) downstream do not change the original `KernelConnectionMetadata`.
- * Hence always clone the `KernelConnectionMetadata` returned by the `kernelSelector`.
- */
-@injectable()
-export class KernelSelector {
```

Comment:
whoo hoo, removed old code.
Last month this was all broken, this was displaying our old kernel picker UI and didn't work.